### DISCUSSION TITLE
How to search *.h / *.cpp file ?

### DISCUSSION THREAD

**Discussion by lainegates** [nvim-neo-tree/neo-tree.nvim#1319] [Q&A] [answered]
I just tried neo-tree which is really awsome that it is so smoothly and ui-friendly !

I first tried with a python project, neo-tree works great when searching files / directories.
But when I switch to a c++ projects, all *.h/*.cpp files are disappeared when searching, while other files (*.md/*.jpg/...) are filtered normally.

I search files with "/" for files, "D" for directories in neo-tree window.

Is there any default settings that controls filtering rules?

**Comment by cseickel** [comment, marked as answer]
I have no idea why it would matter, unless the .h/.cpp files are gitignored. It will use `fd` to do the actual search, or `find` if that's not installed. You can try doing the same search using whichever utility you have installed and seeing what is returned.

**Reply by lainegates** [reply]
after I install with `sudo apt install fd-find`， it works!
